src/main/features/response/views/your-details.njk

```diff
@@ -16,30 +16,30 @@
         {% if form.model.type == 'individual' %}
           {%
             set fields = [
-              { label: 'Your full name', name: 'name' }
+              { readOnly: true, label: 'Your full name', value: claim.claimData.defendant.name }
             ]
           %}
           {% set addressSectionHeading = 'Your address' %}
         {% elif form.model.type == 'soleTrader' %}
           {%
             set fields = [
-              { label: 'Your full name', name: 'name' },
-              { label: 'Business name (optional)', name: 'businessName' }
+              { readOnly: true, label: 'Your full name', value: claim.claimData.defendant.name },
+              { readOnly: true, label: 'Business name (optional)', value: claim.claimData.defendant.businessName }
```

Comment:
I wonder if this still make sense to say `(optional)` for read only fields. For me that is the hint for someone who is filling the form. If value not provided then that heading won't be even displayed.

Reply:
will doublecheck

Reply:
confirmed, remove optional with Art